test/access-control.integration.js

```diff
@@ -324,4 +332,23 @@ describe('access control - integration', function() {
       return '/api/accounts-updating/' + this.accountWithReplaceOnPUTfalse.id + '/replace';
     }
   });
+
+  describe('/accounts with upsertWithWhere', function() {
+    var count = 0;
+    before(function() {
+      var roleModel = loopback.getModelByType(loopback.Role);
+      roleModel.registerResolver('$dummy', function(role, context, callback) {
+        process.nextTick(function() {
+          if (context.remotingContext) {
+            count++;
+          }
+          callback && callback(null, false); // Always true
+        });
+      });
+    });
+    lt.beforeEach.givenModel('accountWithReplaceOnPUTtrue');
```

Comment:
@bajtos  Should I create a new model called account under test/fixtures/common/models .   

I see in test/fixtures/common/models, there is already account.json and inside which the "name" is specified as "accountWithReplaceOnPUTtrue" and plural as "accounts-replacing". Please let me know. For now, I have deleted this block, since I have already added tests for "users" model and "banks" model respectively : 

lt.it.shouldBeDeniedWhenCalledAnonymously('POST', '/api/users/upsertWithWhere');
lt.it.shouldBeDeniedWhenCalledUnauthenticated('POST', '/api/users/upsertWithWhere');
lt.it.shouldBeDeniedWhenCalledByUser(CURRENT_USER, 'POST', '/api/users/upsertWithWhere');

lt.it.shouldBeDeniedWhenCalledAnonymously('POST', '/api/banks/upsertWithWhere');
lt.it.shouldBeDeniedWhenCalledUnauthenticated('POST', '/api/banks/upsertWithWhere');
lt.it.shouldBeDeniedWhenCalledByUser(CURRENT_USER, 'POST', '/api/banks/upsertWithWhere');